src/utils/switchToNetwork.ts

```diff
@@ -32,8 +33,8 @@ export async function switchToNetwork({ library, chainId }: SwitchNetworkArgumen
           {
             chainId: formattedChainId,
             chainName: info.label,
-            rpcUrls: [info.addNetworkInfo.rpcUrl],
-            nativeCurrency: info.addNetworkInfo.nativeCurrency,
+            rpcUrls: [INFURA_NETWORK_URLS[chainId]],
```

Comment:
these are deliberately not the same

we don't want our infura urls in the user's metamask or wallet, because we can change the ids at any time. we should use one of the public endpoints provided by the network